src/main/java/umc/spring/service/MissionService/MissionQueryServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package umc.spring.service.MissionService;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import umc.spring.apiPayload.exception.handler.MissionHandler;
+import umc.spring.converter.MissionChallengeConverter;
+import umc.spring.converter.MissionConverter;
+import umc.spring.domain.Store;
+import umc.spring.domain.User;
+import umc.spring.domain.UserMission;
+import umc.spring.domain.mapping.Mission;
+import umc.spring.repository.MissionRepository.MissionRepository;
+import umc.spring.repository.StoreRepository.StoreRepository;
+import umc.spring.repository.UserMissionRepository.UserMissionRepository;
+import umc.spring.repository.UserRepository.UserRepository;
+import umc.spring.web.controller.dto.MissionChallenge.MissionChallengeRequestDTO;
+import umc.spring.web.controller.dto.MissionChallenge.MissionChallengeResponseDTO;
+import umc.spring.web.controller.dto.MissionDTO.MissionRequestDTO;
+import umc.spring.web.controller.dto.MissionDTO.MissionResponseDTO;
+
+import static umc.spring.apiPayload.code.status.ErrorStatus.*;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class MissionQueryServiceImpl implements MissionQueryService {
+
+    private final MissionRepository missionRepository;
+    private final StoreRepository storeRepository;
+    private final UserRepository userRepository;
+    private final UserMissionRepository userMissionRepository;
+
+    @Override
+    @Transactional
+    public MissionResponseDTO.JoinResultDTO JoinDTO(MissionRequestDTO.JoinAddMissionDTO request) {
+
+        Store store = storeRepository.findById(request.getStoreId())
+                .orElseThrow(()-> new MissionHandler(Mission_STORE_NOT_FOUND));
```

Comment:
예시로 이 부분을 해당 미션 서비스에서 레포지토리에 접근하는 것이 아닌,

StoreQueryService에 다음과 같이
```
public Store getStore(Long storeId) {
        return storeRepository.findById(storeId())
                .orElseThrow(()-> new MissionHandler(Mission_STORE_NOT_FOUND));
}
```

이렇게 함수를 만들어주고 mission 서비스에서는 이 store 서비스 함수를 호출하는 방식으로 하는 것이 좋습니다.
```suggestion
        Store store = storeQueryService.getstore(request.getStoreId());
```